### DISCUSSION TITLE
JS backend script to archive done tasks from Task Manager

### DISCUSSION THREAD

**Discussion by jwhonce** [TriliumNext/Trilium#4081] [Show and tell]
My first attempt at JavaScript and writing a trilium script.  It finds done tasks and marks them as #archived.  It does not delete done tasks or anything destructive. Any suggestions on my JavaScript code or any fun features would be welcomed.

https://github.com/jwhonce/trilium-addons/blob/main/archiveDoneTask/archiveDoneTask.js 

There are still some TODO's but I think it is complete enough to share.

**Comment by meichthys** [comment]
Nice. I've added it to https://github.com/Nriver/awesome-trilium 👍
